src/app/modules/companies/store/companies.reducer.ts

```diff
@@ -98,5 +104,27 @@ function mapToCompany(
           .find(x => x?.name === COMPANY_DYNAMIC_PROPERTIES.boolean)?.value as string | null,
         value => /$true^/i.test(value)
       ),
+      shortTextDictionary: organization.dynamicProperties
+        .find(x => x?.name === COMPANY_DYNAMIC_PROPERTIES.shortTextDictionary)?.value as string | null,
     };
 }
+
+function takeDictionaryItems(organization?: getOrganization_organization |
+  Omit<(
+    updateOrganization_updateOrganization &
+    updateMemberDynamicProperties_updateMemberDynamicProperties
+  ), '__typename'> | null): Record<string, string[]> | null {
```

Comment:
You don't need it there, because you take dictionary items only when you load organization, but not on update org/dyn. props